WeakAuras/Prototypes.lua

```diff
@@ -104,7 +104,9 @@ function WeakAuras.UnitChannelInfo(unit)
   end
 end
 
-
+local constants = {
+  nameRealmFilterDesc = L["Filter formats: 'Name', 'Name-Realm', '-Realm'. Multiple Filters can be separated by ','."],
```

Comment:
For multiple entries it could use the sentence as the tooltip for encounter ids , zones, etc...
"\n\nSupport multiple entries, separated by commas"